### PR TITLE
Command: php => c#

### PR DESCRIPTION
**Pull request by ivandrofly**
TEST: https://dotnetfiddle.net/rodoTS

### CODE REVIEW

**Comment by aloisdg** [comment]
Can you add a link to .netfiddle in comment?

**Comment by ivandrofly** [comment]
> Can you add a link to .netfiddle in comment?

Sure :)

Edited: https://dotnetfiddle.net/rodoTS /cc @aloisdg

**Comment by ivandrofly** [comment]
https://dotnetfiddle.net/rodoTS

File: README.md [hunk 1]
```diff
@@ -1267,87 +1267,102 @@ Wikipedia says
...
+```c#
 // Receiver
-class Bulb {
-    public function turnOn() {
-        echo "Bulb has been lit";
+class Bulb
+{
+    public void TurnOn()
+    {
+        Console.WriteLine("Bulb has been lit");
```
Comment (aloisdg) [inline README.md]: You missed a `!` at the end of `Bulb has been lit`

**Review by aloisdg — CHANGES_REQUESTED** [review]

**Pull request merged**